### ISSUE TITLE
WCAG 2.0 errata missing from WCAG 2.0

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag#920]
The changes outlined in https://www.w3.org/WAI/WCAG20/errata/, were never published as part of [WCAG 2.0](https://www.w3.org/TR/WCAG20/), only appearing as part of WCAG 2.1. This seems like an oversight to me.

I also think it would be helpful to provide information about this in the understanding documents of the SCs that were changed, maybe under an "Errata" heading. Ideally something is also added to WCAG 2.0 / 2.1 to make it clear that those SCs were changed, although I can see why that may be more difficult.

**Comment by alastc** [comment]
This caught me out to, I have a feeling there is a policy reason that @michael-n-cooper might be able to explain?

**Comment by WilcoFiers** [comment]
Any update on this @alastc?

**Comment by patrickhlauke** [comment]
@alastc @iadawn @kfranqueiro can this be closed?

**Comment by kfranqueiro** [comment]
At this point in time (admittedly several years after this was first opened), there are no plans to republish 2.0 normative or informative documents (both of which exist in TR space). The errata are accessible from the Recommendation via a link near the top of the document.
